Checks/PWR068/README.md

```diff
@@ -150,11 +150,55 @@ Factorial of           5 is         120
 ```
 
 > [!NOTE]
-> While you can manually define explicit interfaces using the `interface`
-> construct at the call site, this approach requires duplicating the procedure's
-> definition, increasing the likelihood of errors.
-
-> [!NOTE]
+> The previous example demonstrates how calls to `external` procedures are
+> performed through implicit interfaces. A similar situation arises when
+> calling an implicitly declared procedure, as shown in the following example:
+> 
+> ```fortran
```

Comment:
nit:
```suggestion
> ```fortran {5}
```

Reply:
Done! I've used the opportunity to add relevant highlights to already existing code blocks.